src/config/defaults_cat_general.go

```diff
@@ -28,18 +28,14 @@ var generalRequiredByOriginEngines = []engines.Name{
 	engines.GOOGLE,
 }
 
-var generalPreferredEngines = []engines.Name{
-	engines.ETOOLS, // Not in ByOrigin because it only gives 10 results across a lot of engines that it scrapes from.
-}
+var generalPreferredEngines = []engines.Name{}
 
 var generalPreferredByOriginEngines = []engines.Name{
 	engines.BRAVE,
-	engines.MOJEEK,
```

Comment:
Mojeek gives mostly useless results so there's no need to wait for it in general category.